Analytics/AnalyticsLogging/AnalyticsLogger.swift

```diff
@@ -7,7 +7,9 @@
 //
 
 import Foundation
+#if !COCOAPODS
```

Comment:
This change is caused by cherry-pick the import GraphQL error from another branch, please ignore in this PR.